src/plugins/intel_gna/src/gna_plugin.hpp

```diff
@@ -53,6 +55,7 @@ class GNAPlugin : public InferenceEngine::IInferencePlugin {
     uint32_t activeLayerIndex = 0xffffffff;
     TranspositionInfoMap transpose_inputs_info;
     TranspositionInfoMap transpose_outputs_info;
+    PrePostProcessModels subgraph_cpu_map;
```

Comment:
It isn't clear to me what could a subgraph_cpu_map variable contain, even or maybe especially when connected with the variable type name.

Reply:
renamed